frontend/src/components/spatialChat/chatSettings/ChatSettings.tsx

```diff
@@ -153,6 +154,7 @@ export default function ChatSettings() {
                   onChange={e => setLastRadius(e.target.value)}
                   name='chatRadius'
                   type='text'
+                  className={constants.CUSTOM_PRIORITY_FOCUS_CLASS_FOR_INPUT}
```

Comment:
Adding this class will allow the keyboard inputs to the border-radius to work correctly (especially the arrow keys, 
which I found I useful when changing only 1 value).